src/command.c

```diff
@@ -627,6 +628,32 @@ int cmd_auth(struct command *cmd, struct redis_data *data)
     return CORVUS_OK;
 }
 
+int cmd_time(struct command *cmd)
+{
+	struct timeval tm;
+	int ret = gettimeofday(&tm, NULL);
+	if (ret) {
+		return CORVUS_ERR;
+	}
+	char time_fmt[100];
+	char time_sec[15];
+	char time_us[10];
+	snprintf(time_sec, sizeof(time_sec), "%d", tm.tv_sec);
+	snprintf(time_us, sizeof(time_us), "%d", tm.tv_usec);
```

Comment:
yeah, fix the 2038 year problem altogether
fix the intent of code using space chars
add the error logging

& made a new commit on this

Thank you for your responding